arch/risc-v/include/types.h

```diff
@@ -54,12 +54,21 @@ typedef unsigned char      _uint8_t;
 typedef signed short       _int16_t;
 typedef unsigned short     _uint16_t;
 
+/* We could use `long long` as 64-bit for all ABIs, but that causes
+ * "format warings" beyond arch/risc-v scope. So let's revisit it later.
+ */
+
 #ifdef CONFIG_ARCH_RV64
 typedef signed int         _int32_t;
 typedef unsigned int       _uint32_t;
 
+#ifndef CONFIG_ARCH_RV64ILP32
```

Comment:
should share the same CONFIG_ARCH_RV32 part from line 73-77

Reply:
let me try